webpack.config.js

```diff
@@ -1,6 +1,42 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
+/*
```

Comment:
@SunburtReynolds I think the `context` config could fix this: https://webpack.js.org/configuration/entry-context/#context

Reply:
@narinluangrath unfortunately, I tried `context` but could not get it to work! If you have the time to try it out and find a particular configuration that doesn't require all the absolute paths to dependencies, I will absolutely take it 🙏

Reply:
Aw man, that is really surprising. I'll try it out tonight.